src/olympia/amo/tests/__init__.py

```diff
@@ -614,6 +620,21 @@ def make_addon_promoted(cls, addon, group_id, approve_version=False, apps=None):
             addon.approve_for_version(addon.current_version)
         return promotions
 
+    @property
+    def promoted_groups(self):
+        return {
+            id: PromotedGroup.objects.get(group_id=id) for id in PROMOTED_GROUPS_BY_ID
+        }
+
+    def make_promoted(self, addon, group_ids=None, apps=None, approve_version=False):
+        group_ids = group_ids or []
+        for group_id in group_ids:
+            self.make_addon_promoted(
+                addon=addon, group_id=group_id, apps=apps, reset=False
+            )
+        if approve_version:
+            addon.approve_for_version()
```

Comment:
I don't think you need these in `TestCase` - they're only used in one test class.

(I'm not sure I'd have added `make_promoted` either, when it's only saving you from having to do 
```python
self.make_addon_promoted(addon=addon, group_id=PROMOTED_GROUP_CHOICES.RECOMMENDED, reset=False)
self.make_addon_promoted(addon=addon, group_id=PROMOTED_GROUP_CHOICES.LINE, reset=False)
```
i.e. 1 line per extra group but 🤷‍♂️ )